Add instructions for usage as npm package

alex292 opened the pull request:
PTAL, thanks!

kbalazs-ms commented:
So now we have a single NPM package generated (thank you for that 👍) and deployed to the NPM registry, but how is this gonna be maintained? 
(released with every new version of the dictation_support library)

I would assume that it was manually generated, because I don't see an action/workflow step added, or anything that ties the NPM package release into the release pipeline. 
For future maintenance and support, I would love to see this as the result of a new version release, automatically created and uploaded.
 - We can already see how we have a 1.0.2 released as an NPM package, but no released 1.0.2 version of the library on the releases page. I think these two should be tightly linked together and not handled manually.

Also, small thing but probably useful. The released NPM package doesn't have a the GitHub repository set, so its not pointing to this repository on the NPM site, and not listed here as an existing package.

Thanks again for taking the time to improve the library's usability!

rpriewasser-sps reviewed (changes requested):
Can we please address Balazs´ requests made in his [latest comment ](https://github.com/GoogleChromeLabs/dictation_support/pull/10#issuecomment-1428073404) regarding future maintenance and support?

alex292 commented:
I have added the repository link to the package.json so it properly links back from npmjs.com.

I have never set up automated npm publishing, but it surely is feasible with an automated github workflow action and a generated access token. I don't have the cycles to take care of that at the moment and given that I/we don't expect this package to change on a regular basis, I think manually running `npm publish` is a reasonable alternative for now.
@rpriewasser-sps could you also create an account on npmjs.com and then I can add you as owner there as well so you can also publish it yourself. Feel free to add anyone else required.

Since I have switched teams internally, I would expect you (and your team) to handle long term maintenance of this package.

rpriewasser-sps commented:
Thanks @alex292, I´ve created an account on npmjs: rpriewasser-sps

alex292 commented:
I've added you as maintainer to the npm package
